q2_qemistree/_plot.py

```diff
@@ -125,6 +166,11 @@ def plot(output_dir: str, table: biom.Table, tree: NewickFormat,
     itol_uploader.add_file(target)
     itol_uploader.add_file(label_fp)
     itol_uploader.add_file(color_fp)
+    if grouped_table:
```

Comment:
Perhaps this works for BIOM, but in recent Pandas versions it will throw a warning: "it is ambiguous to determine the True/False of a DataFrame..."

Reply:
In fact, it would raise an error & terminate if `table` were a pandas df

Reply:
It might be better to be verbose about this and check `if grouped_table is not None:` since that's the "default" value.

Reply:
👍